data/assets/scene/digitaluniverse/constellations.asset

```diff
@@ -105,7 +105,7 @@ local show_constellations = {
     openspace.fadeIn("Scene.Constellations.Renderable")
   ]],
   Documentation = "Shows all the constellations lines",
-  GuiPath = "/Constellations/Lines",
+  GuiPath = "/Night Sky/Constellations/Lines",
```

Comment:
I'd not make this change since the constellation lines are also useful outside of the solar system.  And if _that_ is the night's sky, then everything is, making the distinction less useful

Reply:
I disagree. I think putting them in the Night Sky folder is the way to go since constellations are something from the night sky on earth. 

Most users who are giving a night sky show will want to show the constellations.